compiler/vmconv.nim

```diff
@@ -13,6 +13,13 @@ proc fromLit*(a: PNode, T: typedesc): auto =
     type Ti = elementType(T)
     for ai in a:
       result.incl Ti(ai.intVal)
+  elif T is seq:
+    result = default(T)
+    let length = a.len
+    result.setLen(length)
+    type Ti = elementType(T)
+    for i in 0 ..< length:
+      result[i] = Ti(a[i].intVal)
```

Comment:
`set` was special in that it only supports Ordinal, but seq should be more general and intVal won't work for all `Ti`; add at least a comment saying:
```nim
# xxx support other `a[i].kind`
```
(this shouldn't be too hard to support, at least floatVal or even `node`, recursively)

Reply:
could be done in the future